src/scenes/home/press/civicXBadge/civicXBadge.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import civicX from '../../../../images/civic-x.png';
+import styles from './civicXBadge.css';
+
+function CivicXBadge() {
+  return (
+    <div>
+      <a href="http://cvcx.org/veterans-solutions-lab/">
+        <img
+          alt="Operation Code is a graduate of the CivicX Accelerator."
+          className={styles.civic}
+          src={civicX}
+        />
+      </a>
+    </div>
+  );
+}
+
+export default CivicXBadge;
```

Comment:
Instead of using the `function` syntax, so far we've typically been using ES2015 arrow functions assigned to `const` variables within our React components - see staffCard.js or teamCard.js for examples.

Reply:
great suggestion - updated!